Packs/Rapid7_InsightIDR/Integrations/Rapid7_InsightIDR/1_09b2a1a3-aa1e-4e41-b0c4-6bffaf32e341

```diff
@@ -0,0 +1,2107 @@
+{
```

Comment:
Why were this file and `Packs/Rapid7_InsightIDR/Integrations/Rapid7_InsightIDR/1_75618cae-93e7-4a58-887f-7b4d56af5667` file added?